alibi_detect/cd/base.py

```diff
@@ -203,8 +209,8 @@ def score(self, x: Union[np.ndarray, list]) -> Tuple[float, float, np.ndarray, n
         pass
 
     def predict(self, x: Union[np.ndarray, list],  return_p_val: bool = True,
-                return_distance: bool = True, return_probs: bool = True) \
-            -> Dict[Dict[str, str], Dict[str, Union[int, float]]]:
+                return_distance: bool = True, return_probs: bool = True, return_model: bool = True) \
```

Comment:
What is the reason for returning the model? To understand what model exactly resulted in the drift predictions?

Reply:
Was useful for implementing `SpotTheDiffDrift` detector as a wrapper around the `ClassifierDrift` detector, but thought could be useful more generally too.